src/enums.ts

```diff
@@ -39,3 +39,9 @@ export enum DualKpiHoverDataPercentType {
     firstDate = "First date",
     previousDate = "Previous date"
 }
+
+export enum PercentType {
+    lastDate = <any>"lastDate",
```

Comment:
why we need <any> here in this enum?

Reply:
It looks like the old code. We don't need `<any>` here. I removed it.